src/pre_defined_classes/simple_programs_taking_numeric_user_input/hard/q10/NewSalaryCalculator.java

```diff
@@ -0,0 +1,39 @@
+package pre_defined_classes.simple_programs_taking_numeric_user_input.hard.q10;
+/*
+A company called HappyWorkLife has decided to give everybody a raise with an increase of 5% for each year of
+their service. The raise is only given to those who have worked at least a year or more,
+and the raise is calculated annually. Make a program that asks the user for the salary and the number of years worked
```

Comment:
If the raise is calculated annually, then the code should be changed to accept an integer value for the year